### ISSUE TITLE
Feature request : simplify map background selection

### ISSUE THREAD

**Issue by pbalyx** [umap-project/umap#1034]
When user wants to change the map background, he is faced to a huge list of maps. Most of them are probably never or very rarely used. It would be nice if the map editor could select a subset from this list that would be the only one presented later when one wants to select a base map. This would be an improvement for the editor himself but especially when the map is intended to be used by a large public.

Note : issues #272 and #986 still opened covered roughly the same request but they have never been answered

**Comment by archie83** [comment]
Hi,
I have the same problem ! I think it would be a big enhancement for usability.

**Comment by yohanboniface** [comment]
There is now a switcher with only few backgrounds, and a plus button get the full list for the adventurers.

**Comment by pbalyx** [comment]
I waited a few days to make sure I had the latest release but I think this problem has not been solved.
What I said in the first post:
"The user... is presented with a huge list of maps."
and I proposed:
"It would be nice if the map editor could select a subset of this list that would be the only one presented later..."
Adding a plus button to the tile selector did not solve these problems.
- the default selection list is still not set by the map editor
- and when the user clicks the plus button, he is still presented with the full list.
For example, I tried to have at the same time the tiles Plan_IGN and IGN_Image aérienne in the selector that is seen by the user but it is not possible. When I add the second one in editor mode, the first one disappears and user has to seek for it by pressing the plus button.

note: issue #272 which had roughly the same request (in 2016...) has been moved to issue #1814 which proposes exactly the same thing as me. Fortunately, this last issue is still open.
